docs/resources.rst

```diff
@@ -0,0 +1,81 @@
+Resources
+=========
+
+Tutorials
+---------
+
+Django Debugging Tutorial
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Originally presented as an in-person workshop at DjangoCon US 2022, this
+tutorial by **Tim Schilling** covers debugging techniques in Django. Follow
+along independently using the slides and GitHub repository.
+
+* `Read the blog post <https://www.better-simple.com/django/2022/11/18/django-debugging-tutorial/>`__
+* `Follow along with the GitHub repo <https://github.com/tim-schilling/debug-tutorial/>`__
+* Last updated: February 13, 2025.
+* Estimated time to complete: 1-2 hours.
```

Comment:
I really like that you put the watch time and estimated time to complete here. That's a fantastic addition!